weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Here, I propose to change the var name <code>workflow_ids</code> to <code>response</code> . </p><p> </p><p>Because:</p><ul><li>It contains a list of objects (TransformationResponse)</li><li>Each object includes the op name &amp; workflow ID</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>